src/server.rs

```diff
@@ -1254,6 +1262,23 @@ impl ServerStats {
         );
         set_lang_stat!(stats_vec, self.cache_hits, "Cache hits");
         set_lang_stat!(stats_vec, self.cache_misses, "Cache misses");
+
+        set_stat_precentage!(stats_vec, self.cache_hits.all(), self.cache_misses.all() + self.cache_hits.all(), "Cache hits rate");
+
+        let mut sorted_stats_hits: Vec<_> = self.cache_hits.counts.iter().collect();
+        sorted_stats_hits.sort_by_key(|v| v.0);
+        let mut sorted_stats_misses: Vec<_> = self.cache_misses.counts.iter().collect();
+        sorted_stats_misses.sort_by_key(|v| v.0);
+
+        for ((lang_hits, &count_hits), (lang_misses, &count_misses)) in sorted_stats_hits.into_iter().zip(sorted_stats_misses.into_iter()) {
```

Comment:
I don't think this would work in the case where there are only hits or misses for a given language. You'd need to enumerate the intersection of all languages in hits and misses.